cmd/kpod/load.go

```diff
@@ -95,14 +95,18 @@ func loadCmd(c *cli.Context) error {
 		output = os.Stdout
 	}
 
+	options := libpod.CopyOptions{
+		SignaturePolicyPath: c.String("signature-policy"),
+	}
+
 	src := libpod.DockerArchive + ":" + input
-	if err := runtime.PullImage(src, false, c.String("signature-policy"), output); err != nil {
+	if err := runtime.PullImage(src, false, options, output); err != nil {
 		src = libpod.OCIArchive + ":" + input
 		// generate full src name with specified image:tag
 		if image != "" {
 			src = src + ":" + image
 		}
-		if err := runtime.PullImage(src, false, "", output); err != nil {
+		if err := runtime.PullImage(src, false, options, output); err != nil {
```

Comment:
Should the "false" flag and output be part of copy options?

Reply:
The false is for the all-tags flag, which is hidden since it cannot be implemented just yet. Should I remove that flag completely?
output can be part of CopyOptions, what do you say?

Reply:
Yes remove it.  Is this something docker can do?
Output is already in copy options, I believe?

Reply:
yes, docker has that flag

Reply:
And CopyOptions doesn't have output. should I add it?

Reply:
Is this just reportwriter?

Reply:
yes, it is reportWriter.